Update dev-environments.md (#13906)
docker/docker.github.io: desktop/dev-environments.md

>
> When cloning a Git repository using SSH, ensure you've added your SSH key to the ssh-agent. To do this, open a terminal and run `ssh-add <path to your private ssh key>`.

> **Note**
>

> > If you have enabled the WSL2 integration in Docker Desktop for Windows, make sure you have an SSH agent running in your WSL2 distribution.

Reviewer: ```suggestion
> If you have enabled the WSL 2 integration in Docker Desktop for Windows, make sure you have an SSH agent running in your WSL 2 distribution.
```

> The expansible section will contain a piece of script that you shall append to the bottom of your `~/.profile`. Then, re-open a new WSL2 terminal afterwards.

Reviewer 2: instead of mentioning only `~/.profile` file, can we mention instead:
> your profile file (ie : `~/.profile`, `~/zshrc`, ...)`
And isn't a `source` command enough? do we really need to re-open a terminal?

Author: Added more profile files, thanks

<div class="panel panel-default">
    <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapse-wsl2-ssh" style="cursor: pointer">
    How to start an SSH agent in WSL2
    <i class="chevron fa fa-fw"></i></div>
    <div class="collapse block" id="collapse-wsl2-ssh">

>     If your WSL2 distribution doesn't have an `ssh-agent` running you can append this script to the bottom of your profile file (ie : ~/.profile, ~/zshrc, ...).

Reviewer 2: ```suggestion
    If your WSL2 distribution doesn't have an `ssh-agent` running you can append this script to the bottom of your profile file (ie : ~/.profile, ~/.zshrc, ...).
```

>     If your WSL2 distribution doesn't have an `ssh-agent` running you can append this script to the bottom of your profile file (ie : ~/.profile, ~/.zshrc, ...).

Reviewer: ```suggestion
    If your WSL 2 distribution doesn't have an `ssh-agent` running, you can append this script to the bottom of your profile file (ie : ~/.profile, ~/.zshrc, ...).
```

> > If you have enabled the WSL2 integration in Docker Desktop for Windows, make sure you have an SSH agent running in your WSL2 distribution. Otherwise, click on the expansible section below to learn how to start a new SSH agent in case there is not one running already.
> The expansible section will contain a piece of script that you shall append to the bottom of your `~/.profile`. Then, re-open a new WSL2 terminal afterwards.
> 
> <div class="panel panel-default">
>     <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapse-wsl2-ssh" style="cursor: pointer">
>     How to start an SSH agent in WSL2
>     <i class="chevron fa fa-fw"></i></div>
>     <div class="collapse block" id="collapse-wsl2-ssh">
> <pre><code>

Reviewer 3: ```suggestion
> If you have enabled the WSL2 integration in Docker Desktop for Windows, make sure you have an SSH agent running in your WSL2 distribution.

<div class="panel panel-default">
    <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapse-wsl2-ssh" style="cursor: pointer">
    How to start an SSH agent in WSL2
    <i class="chevron fa fa-fw"></i></div>
    <div class="collapse block" id="collapse-wsl2-ssh">
    If your WSL2 distribution doesn't have an `ssh-agent` running you can append this script to the bottom of your `~/.profile` file.
<pre><code>
```

Reviewer 4: +1 to simplify this as much as possible. 
Maybe we can still mention something like :
```
make sure you have an SSH agent running in your WSL2 distribution, for example adding it to your profile file (ie : ~/.profile, ~/zshrc, ...).
```
